.github/workflows/build.yml

```diff
@@ -6,6 +6,7 @@ env:
   CROSS_DEBUG: 1
 
 on:
+  workflow_dispatch:
```

Comment:
Makes no sense because there is additional parameter for source code checkout on https://github.com/tree-sitter/tree-sitter/pull/2178/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L42
The main CICID workflow pretty quickly calls this workflow with correct additional inputs and should be used instead as a main entry point and it has `workflow_dispatch` trigger.

Reply:
`workflow_dispatch` is for manually triggering a workflow; we've found it quite useful on occasion, and it doesn't hurt otherwise. but here it's mainly left over from my testing on my fork; feel free to remove, of course.